<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SingaravelanNandakumars-MacBook-Pro.local" name="LegacyApp" tests="3" failures="1" timestamp="30 Jan 2018 15:41:43 GMT" time="22.772" errors="0">
  <testcase name="orderItemWithSavedCard" time="3.244" classname="co.bytemark.test.orders.OrdersRegressionCases">
    <failure type="java.lang.AssertionError" message="expected [This credit card is invalid or was not found.  Please supply valid user_uuid along with credit_card_uuid and private_key, or credit_card identifiers and try ag] but found [This credit card is invalid or was not found.  Please supply valid user_uuid along with credit_card_uuid and private_key, or credit_card identifiers and try again.]">
      <![CDATA[java.lang.AssertionError: expected [This credit card is invalid or was not found.  Please supply valid user_uuid along with credit_card_uuid and private_key, or credit_card identifiers and try ag] but found [This credit card is invalid or was not found.  Please supply valid user_uuid along with credit_card_uuid and private_key, or credit_card identifiers and try again.]
	at co.bytemark.helper.Then.assertEquals(Then.java:48)
	at co.bytemark.test.orders.OrdersRegressionCases.orderItemWithSavedCard(OrdersRegressionCases.java:55)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- orderItemWithSavedCard -->
  <testcase name="orderItemWithSavedCard" time="8.119" classname="co.bytemark.test.orders.OrdersRegressionCases"/>
  <testcase name="orderItemWithSavedCard" time="5.234" classname="co.bytemark.test.orders.OrdersRegressionCases"/>
</testsuite> <!-- LegacyApp -->
